rust/src/lib.rs

```
@@ -0,0 +1,74 @@
+#[macro_use(debug)]
```

Clippy (1.96.0)

[warning] unused `#[macro_use]` import

(warning)